src/simple-pg/pool.js

```diff
@@ -0,0 +1,101 @@
+import { Pool } from 'pg';
+import debug from 'debug';
+const log = (...text) => debug(`graffy:pg:${text.shift()}`)(text);
+
+const pgPool = {};
+pgPool.connect = (config) => {
+  const pool = new Pool(config);
+  pgPool.setPool(pool);
+};
+
+pgPool.setPool = (pool) => {
+  // the pool will emit an error on behalf of any idle clients
+  // it contains if a backend error or network partition happens
+  pool.on('error', (err, _) => {
+    log('error', 'Unexpected error on idle client', err);
+    process.exit(-1);
+  });
+  pgPool.pool = pool;
+};
+
+pgPool.setClient = (client) => {
+  pgPool.client = client;
+};
```

Comment:
This also seems unused, and can be removed.

Reply:
actually, we are only using `setClient, however since this is a lib so I open up for various ways to connect to graffy-pg. Anyway, this is a simple version so we can only keep `setClient`